### PR TITLE
docs: add explanation to fork StackBlitz before save

### PR DESCRIPTION
**Pull request by enekonieto**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by enekonieto** [comment]
I signed it!

2 de julio de 2019 12:17, "googlebot" <notifications@github.com (mailto:notifications@github.com?to=%22googlebot%22%20<notifications@github.com>)> escribió:
	Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA). 

	https://github.githubassets.com/images/icons/emoji/unicode/1f4dd.png">📝Please visit https://cla.developers.google.com/ (https://cla.developers.google.com/) to sign. 

	Once you've signed (or fixed any issues), please reply here (e.g. I signed it!) and we'll verify it. 
------------------------------------
What to do if you already signed the CLA
Individual signers
	* It's possible we don't have your GitHub username or you're using a different email address on your commit. Check your existing CLA data (https://cla.developers.google.com/clas) and verify that your email is set on your git commits (https://help.github.com/articles/setting-your-email-in-git/).
Corporate signers
	* Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to go/cla#troubleshoot (http://go/cla#troubleshoot) (Public version (https://opensource.google.com/docs/cla/#troubleshoot)). 
	* The email used to register you as an authorized contributor must be the email used for the Git commit. Check your existing CLA data (https://cla.developers.google.com/clas) and verify that your email is set on your git commits (https://help.github.com/articles/setting-your-email-in-git/). 
	* The email used to register you as an authorized contributor must also be attached to your GitHub account (https://github.com/settings/emails). 
	https://github.githubassets.com/images/icons/emoji/unicode/2139.png">ℹ️Googlers: Go here (https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fangular%2Fangular%2Fpull%2F31384) for more info.

File: aio/content/start/index.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ We've seeded this particular app with a top bar&mdash;containing the store name
 <div class="callout is-helpful">
 <header>StackBlitz tips</header>
 
-* Log into StackBlitz, so you can save and resume your work. If you have a GitHub account, you can log into StackBlitz with that account. 
+* Log into StackBlitz, so you can save and resume your work. If you have a GitHub account, you can log into StackBlitz with that account. Before being able to save you must fork the project using the Fork button at the top left.
```
Comment (kapunahelewong) [inline aio/content/start/index.md]: How about this rephrasing?

> In order to save your progress, first fork the project using the Fork button at the top left, then you'll be able to save your work to your own StackBlitz account by clicking the Save button.

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
Hi @enekonieto! Thanks very much for this improvement! I have just one suggestion.

**Comment by kapunahelewong** [comment]
PS: You'll also want to amend your commit message so that it meets the guidelines. Try something like this to get it to pass:

> docs: add explanation to fork StackBlitz before save

File: aio/content/start/index.md (hunk 1 above)
Comment (enekonieto) [reply aio/content/start/index.md]: I'm not a native english speaker so I suppose your wording is much better than mine. Updated!

Comment (kapunahelewong) [reply aio/content/start/index.md]: Wow, you sound native in this sentence. :) I never would have known. I just recommend the changes for tone. Thanks again for adding this!

**Comment by kapunahelewong** [comment]
Hi @enekonieto! Thanks for incorporating the updates. Circleci is failing because of that one commit message (the first one). Could you squash to one commit (which you'd need to do anyway) and then use your new commit message (`docs: rephased explanation to fork before save`)? After that, once it's all green, would you please tag me?

**Comment by kapunahelewong** [comment]
Thanks @enekonieto! 

Didn't you already sign the CLA? Hm.... GitHub has been acting a little flaky the last few minutes....I restarted the test, so hopefully it will sort itself out. In the process though, I noticed a typo in the commit message. I think you intended `rephased` to be `rephrased`, right? In case you need the command, you can fix this with a `git commit --amend`.

**Comment by enekonieto** [comment]
Done! I used another email account in previous commit, that's why the bot complained about CLA. I copied your message, it sounded strange to me (rephased), but as said, i'm not native english speaker.

18 de julio de 2019 20:19, "Kapunahele Wong" <notifications@github.com (mailto:notifications@github.com?to=%22Kapunahele%20Wong%22%20<notifications@github.com>)> escribió:
	Thanks @enekonieto (https://github.com/enekonieto)! 

	Didn't you already sign the CLA? Hm.... GitHub has been acting a little flaky the last few minutes....I restarted the test, so hopefully it will sort itself out. In the process though, I noticed a typo in the commit message. I think you intended rephased to be rephrased, right? In case you need the command, you can fix this with a git commit --amend.

**Comment by kapunahelewong** [comment]
Oooh! My apologies! I am unusually skilled at the sneaky typo. 😁

**Review by kapunahelewong — APPROVED** [review]

**Pull request closed without merging**